fs/accounting/stats.go

```diff
@@ -624,11 +624,29 @@ func (s *StatsInfo) RemoveTransfer(transfer *Transfer) {
 	s.mu.Unlock()
 }
 
-// PruneTransfers makes sure there aren't too many old transfers
+// PruneAllTransfers removes all finished transfers.
```

Comment:
@zimnx this function is not used, it should be used in cleanup flow IMO.

Reply:
right, I screwed this when I was resolving conflicts. On it.

Reply:
done